### PR TITLE
COMMERCE-1453 Remove SF suppressions for CommerceRegionsStarter and CommerceShippingOriginLocator

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test

**Comment by marco-leo** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

File: commerce-service/src/main/java/com/liferay/commerce/internal/starter/BaseCommerceRegionsStarter.java [hunk 1]
```diff
@@ -33,7 +33,11 @@
...
 	@Override
-	public void start(ServiceContext serviceContext) throws Exception {
+	public void start(long userId) throws Exception {
+		ServiceContext serviceContext = new ServiceContext();
+
+		serviceContext.setUserId(userId);
+
 		CommerceCountry commerceCountry =
 			commerceCountryLocalService.fetchCommerceCountry(
 				serviceContext.getCompanyId(), getCountryIsoCode());
```
Comment (ethanbustad) [inline commerce-service/src/main/java/com/liferay/commerce/internal/starter/BaseCommerceRegionsStarter.java]: `serviceContext.getCompanyId()` will return `0` here as is. needs a little revamp

**Review by ethanbustad — CHANGES_REQUESTED** [review]

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

@marco-leo @ethanbustad Please re-review my last 2 commits

**Comment by alecsloan** [comment]
ci:test

**Comment by ethanbustad** [comment]
☝️ @alecsloan I think this failure could be from commit https://github.com/liferay/com-liferay-commerce/pull/1513/commits/7d90eae7669df6209477e565a04aa337f62d8f0e, though I'm not sure exactly why it's failing atm

**Pull request merged**